### ISSUE TITLE
NVDA unable to read list items and notification area part of Windows 8.1. 

### ISSUE THREAD

**Issue by q9alpha** [nvaccess/nvda#5625]
Unable to provide steps to reproduce, as it only happened suddenly. NVDA basically doesn't read the list items on say, the desktop, and in the case when uninstalling a program, throws error messages and error tones. The log segment is provided for examination. 
  The only strange part about the issue is that when navigating in "File Explorer", list items are reported correctly. 

NVDA Version: Next - 13006,7139cad
OS: Windows 8.1 Professional 64-bit. 

[log.txt](https://github.com/nvaccess/nvda/files/67979/log.txt)

**Comment by jcsteh** [comment]
This is an instance of #3339.
